cmd/trade.go

```diff
@@ -860,6 +885,12 @@ func deleteAllOffersAndExit(
 			return
 		}
 
+		e = tracker.SendDeleteEvent(l, true)
+		if e != nil {
+			l.Info(fmt.Sprintf("Sending deletion data event failed with error: %v\n", e))
```

Comment:
wrap and throw error.
can you use the word `metric` in all the log messages you've added? (example: delete event metric)
that will be very helpful in sifting through logs for these events